test/e2e/common.go

```diff
@@ -205,6 +205,89 @@ func filterNodeCondition(conditions []corev1.NodeCondition, conditionType corev1
 	return filtered
 }
 
+// listBareMetalHosts logs the names, provisioning status, consumer and power status
+// of all BareMetalHosts matching the opts. Similar to kubectl get baremetalhosts.
+func listBareMetalHosts(ctx context.Context, c client.Client, opts ...client.ListOption) {
+	bmhs := bmov1alpha1.BareMetalHostList{}
+	Expect(c.List(ctx, &bmhs, opts...)).To(Succeed())
+	Logf("Listing BareMetalHosts:")
+	Logf("Name	Status	Consumer	Online")
+	Logf("---------------------------------------------------------------------------------")
```

Comment:
I do not like that every line starts with INFO: what do you think if we removed it from Logf or just create  customized log function for the resources profiling

Reply:
Also I see sometimes the output are not well formatted! what if we used utility to print formatted tables such https://github.com/chrusty/go-tableprinter

Reply:
Thanks for the suggestions! I was also thinking about something to make the tables nicer but didn't want to complicate it right now. Maybe we can do it in a followup?
I'm fine with removing INFO also, but should we do it for all or custom for only this? Let's see what others think.